Simperium/src/main/java/com/simperium/client/Bucket.java

```diff
@@ -775,6 +792,22 @@ public void removeOnBeforeUpdateObjectListener(OnBeforeUpdateObjectListener<T> l
         onBeforeUpdateListeners.remove(listener);
     }
 
+    public void addOnSyncObjectListener(OnSyncObjectListener<T> listener) {
+        onSyncListeners.add(listener);
+    }
+
+    public void removeOnSyncObjectListener(OnSyncObjectListener<T> listener) {
+        onSyncListeners.remove(listener);
+    }
+
+    public void addLocalQueueChangeListener(OnLocalQueueChangeListener<T> listener) {
+        onLocalQueueChangeListeners.add(listener);
+    }
+
+    public void removeLocalQueueChangeListener(OnLocalQueueChangeListener<T> listener) {
+        onLocalQueueChangeListeners.remove(listener);
+    }
```

Comment:
```suggestion
    public void addOnLocalQueueChangeListener(OnLocalQueueChangeListener<T> listener) {
        onLocalQueueChangeListeners.add(listener);
    }

    public void removeOnLocalQueueChangeListener(OnLocalQueueChangeListener<T> listener) {
        onLocalQueueChangeListeners.remove(listener);
    }
```

The other methods to add and remove the listeners have "On" in the name.  We can add that to the `addLocalQueueChangeListener` and `removeLocalQueueChangeListener` methods for consistency.